inst/ms/aggreCAT.qmd

```diff
@@ -169,16 +201,17 @@ datasets collected using such elicitation methods.
 
 We begin by formulating the problem of mathematically aggregating expert
 judgements. Each method, and its data requirements is summarised in
-table X (cross-ref). We also briefly summarise package datasets, which
-were collected by the repliCATS project. By first describing the
-datasets before describing the aggregation methods in detail, we aim to
-provide a grounded understanding of the different outputs of expert
-elicitation using the repliCATS IDEA protocol, and the inputs available
-to the aggregation functions.
+Table \ref{tbl-method-summary-table}. Before outlining key aggregation methods, we
```

Comment:
Convention is usually to refer to figures/tables in the order they appear in the paper, right? So this should get listed as Table 1 and shown earlier in the manuscript?

Reply:
I think this is a LaTeX thing where the table was so big it got moved to the end of the paper rather than being held in place... 
- [ ] @egouldo to look into either fixing the number of the methods summary table or else ensure table appears in-place rather than being forced to end of paper.

Reply:
#38

Reply:
Have moved table to end to match numbering until #38 fixed. Resolving thread now.